tests/test_rq_job_utils.py

```diff
@@ -11,22 +12,39 @@
 
 class TestRQJobUtils:
     """Test RQ-related utilities."""
+
     @staticmethod
     def test_launch_simulation_from_rq_job():
-        """Assure the launch_simulation_from_rq_job can successfully launch a simulation."""
-        try:
-            launch_simulation_from_rq_job(
-                scenario=zlib.compress(pickle.dumps("default_2a")),
-                settings={
-                    "duration": duration(days=1),
-                    "slot_length": duration(hours=1),
-                    "tick_length": duration(minutes=6)
-                },
-                events=None,
-                aggregator_device_mapping="null",
-                saved_state=zlib.compress(pickle.dumps(None)),
-                job_id="TEST_SIM_RUNS",
-                connect_to_profiles_db=False
-            )
-        except Exception as exc: # noqa
-            pytest.fail(exc)
+        """Assure the launch_simulation_from_rq_job can successfully launch a simulation.
+        The launch_simulation_from_rq_job is ran in another process to ensure that its changes on
+        constant settings will not cause other tests to fail.
```

Comment:
I'm sort of sorry that we didn't manage to find an easier way to avoid this. I think that in the future we would benefit from avoiding globals as much as possible, to avoid convoluted situations like this.
Thanks for finding this way to overcome the testing issue 👍